lib/ronin/app/cli.rb

```diff
@@ -104,10 +104,18 @@ def run
         Dir.chdir(ROOT)
 
         begin
-          unless is_redis_running?
-            log_info "Starting Redis server ..."
-            pids << start_redis
-            sleep 1
+          if is_valkey_installed?
+            unless is_valkey_running?
```

Comment:
I think these two statements can be combined into `if is_valkey_installed? && !is_valkey_running?`.